aalondon/static/js/components/MeetingSearchForm.js

```diff
@@ -43,57 +43,58 @@ class MeetingSearchForm extends React.Component {
     event.preventDefault();
   }
   handleDayChange(eventKey, e) {
-  
-    
-      this.props.onDayChange(e.value);
+
+
+    this.props.onDayChange(e.value);
   }
   handleIntergroupChange(eventKey, e) {
     this.props.onIntergroupChange(e.target.innerText);
   }
-  
+
   handleSearchChange(e) {
-   
 
-   let lastSearchLength = this.state.search.length;
+
+    let lastSearchLength = this.state.search.length;
     let currentSearchLength = e.target.value.length;
-    
+
     this.setState({ search: e.target.value });
-    if (e.target.value.length > 2 || (lastSearchLength - currentSearchLength === lastSearchLength && currentSearchLength === 0) ){
+    if (e.target.value.length > 2 || (lastSearchLength - currentSearchLength === lastSearchLength && currentSearchLength === 0)) {
       this.props.onSearchChange(e.target.value);
     }
 
-    
 
- }
 
-  handleSearchKeyDown(e){
-    
+  }
+
+  handleSearchKeyDown(e) {
+
     if (e.keyCode === ENTER_KEY) {
-     
+
       this.props.onSearchEnter(e.target.value);
-  } }
+    }
+  }
+
+  handleTimeChange(eventKey, e) {
 
-  handleTimeChange(eventKey,e) {
-   
     this.props.onTimeChange(e.value);
-  } 
+  }
+
+  handleAccessChange(eventKey, e) {
 
-  handleAccessChange(eventKey,e) {
-  
     this.props.onAccessChange(e.value);
-  } 
-  handleCovidChange(eventKey,e) {
-  
+  }
+  handleCovidChange(eventKey, e) {
+
     this.props.onCovidChange(e.value);
-  } 
+  }
 
   handleClearFilters() {
-    
+
     this.props.onClearFilters();
-  } 
+  }
 
-  handleMeetingTypeChange(eventKey,e){
-   this.props.onMeetingTypeChange(e.value);
+  handleMeetingTypeChange(eventKey, e) {
```

Comment:
I'm a big fan of code tidy-ups, but mixing them with feature implementations in the same commit makes it tricky for me to do code-reviews.

In future, could you please split code tidy-ups & feature implementations into dedicated commits? i.e. a feature impl. should have it's own commit without code tidy ups.

Reply:
Yes I must have done a format doc in vscode